packages/models/src/risk-analysis/riskAnalysis.ts

```diff
@@ -28,6 +28,12 @@ export const RiskAnalysisForm = z.object({
 });
 export type RiskAnalysisForm = z.infer<typeof RiskAnalysisForm>;
 
+export const PurposeRiskAnalysisForm = z.object({
+  ...RiskAnalysisForm.shape,
+  riskAnalysisId: RiskAnalysisId.optional(),
+});
```

Comment:
An easy way to extend an object in zod is to use the `and` method.

```suggestion
export const PurposeRiskAnalysisForm = RiskAnalysisForm.and(
  z.object({
    riskAnalysisId: RiskAnalysisId.optional(),
  })
);
```

Reply:
Updated, thanks!